react/Header/Header.js

```diff
@@ -16,17 +16,22 @@ const employerLinkHref = locale => locale === 'NZ' ?
   'https://talent.seek.co.nz/' :
   'https://talent.seek.com.au/';
 
+const AUTHENTICATED = 'authenticated';
```

Comment:
I looked into exporting these constants but it required a fair bit of boilerplate for the benefit returned (in terms of wiring up @markdalgleish's code above to the PropTypes). I would prefer this to go in consistently for now, but we discuss how to manage this pattern going forward